SAMv1.tex

```diff
@@ -658,6 +659,21 @@ \section{Recommended Practice for the SAM Format}
     should be present.
   \item The {\tt NM} tag should be present.
   \end{enumerate}
+
+\item Circular reference sequences
+
+Mappings that cross the coordinate `join' in circular reference sequences (i.e., those whose {\tt @SQ} headers specify {\tt TP:circular}) may be represented as follows:
+  \begin{enumerate}[label=\arabic*]
+  \item (Preferred)
+As usual {\sf POS} should be between 1 and the {\tt @SQ} header's {\tt LN} value, but {\sf POS} plus the sum of the lengths of {\tt M/=/X/D/N} {\sf CIGAR} operations may exceed {\tt LN}.
+Coordinates greater than~{\tt LN} are interpreted by subtracting {\tt LN} so that bases at $\texttt{LN}+1, \texttt{LN}+2, \texttt{LN}+3, \ldots$ are considered to be mapped at positions $1,2,3,\ldots$; thus each (1-based) position $p$ is interpreted as $((p-1)\bmod\texttt{LN})+1$.%
+\footnote{The impact of this representation on indexing and random access is yet to be explored by implementations.}
+
+  \item
+Alternatively, such alignments may be split across several records: one record representing the initial portion of the segment ending at~{\tt LN}, one representing the final portion starting from~1, and any other records representing additional portions in between spanning the entire reference sequence.
+One record (chosen arbitrarily) is considered primary and the remainder have their {\sf supplementary} flag bit (0x800) set.
```

Comment:
small suggestion based on line 678
```suggestion
One record (chosen arbitrarily) is considered primary and the remainder have their {\sf supplementary} {\sf FLAG}  bit (0x800) set.
```

Reply:
Shrug… I was following the similar lowercase ‘flag bit’ phrasing in footnote 12 and in the _Unmapped reads_ and _Multiple mapping_ sections just above…